src/GoodsList.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
+
+type Props = {
+  goods: string[],
+  isReversed: boolean,
+  typeOfSort: string,
+  lengthMin: number
+};
+
+export const GoodsList: React.FC<Props> = (
+  {
+    goods, isReversed, typeOfSort, lengthMin,
+  },
+) => {
+  let copy = [...goods].filter(good => good.length >= lengthMin);
```

Comment:
does filter mutate the original array?
```suggestion
  let copy = goods.filter(good => good.length >= lengthMin);
```